### PR TITLE
Cart Page done

### PR DESCRIPTION
**Pull request by vanshitamoolchandani**
Cart page gives the checkout option and similar products.

### CODE REVIEW

**Review by sri-rishi — APPROVED** [review]
Looks good to me, can be merged

File: Pages/Cart/styles.css [hunk 1]
```diff
@@ -0,0 +1,97 @@
+@import url('/root.css');
```
Comment (git-0r) [inline Pages/Cart/styles.css:1]: review1

one css file request is failing.

File: Pages/Cart/index.html [hunk 2]
```diff
@@ -0,0 +1,294 @@
...
+                <a class="link" href="https://github.com/vanshitamoolchandani">
+                    <i class="fab fa-github social"></i>
+                </a>
+            </li>
+        </ul>
+        <div class="footer-details">
+            <ul class="component-content list-type-none tu-left">
+                <li class="list-heading">ABOUT</li>
+                <li>About Us</li>
+                <li>Caree&#8377/li>
```
Comment (git-0r) [inline Pages/Cart/index.html]: list tag not closed properly.

File: Pages/Cart/index.html [hunk 3]
```diff
@@ -0,0 +1,294 @@
...
+                </div>
+            </div>
+        </div>
+
+
+
+    </div>
+    <!-- Footer -->
+    <footer>
+        <h4>Created by Vanshita Moolchandani</h4>
```
Comment (git-0r) [inline Pages/Cart/index.html:253]: review2

you can remove those blue lines by using text-decoration: "none" on links.

File: Pages/Cart/index.html [hunk 4]
```diff
@@ -0,0 +1,294 @@
...
+            <a href="/Pages/Wishlist/index.html">
+
+                <button class="btn-icon btn-icon-secondary">
+                    <div class="badge-container">
+                        <i class="badge-icon far fa-heart"></i>
+                        <span class="badge-icon-secondary badge-round badge-icon-num">2</span>
+                    </div>
+                </button>
+            </a>
+            <a href="/Pages/Wishlist/index.html">
```
Comment (git-0r) [inline Pages/Cart/index.html]: cart icon is connected to wishlist page.

**Review by git-0r — CHANGES_REQUESTED** [review]

**Comment by prankurpandeyy** [comment]
looks good , input box will be more attractive if you can give some style

**Review by prankurpandeyy — APPROVED** [review]

**Pull request merged**